src/app/(bot)/botsinfo.tsx

```diff
@@ -0,0 +1,35 @@
+import { Text, View } from "react-native";
+import { useState } from "react";
+import PagerView from "react-native-pager-view";
+import ChatHeader from "@/components/ChatHeader";
```

Comment:
export the ChatHeader from components folder as naming export

use kabab-case file format

example 

headers/chat.header.tsx